addons/mail/static/src/core/common/store_service.js

```diff
@@ -93,19 +73,116 @@ export class Store extends BaseStore {
     odoobotOnboarding;
     users = {};
     internalUserGroupId = null;
-    registeredImStatusPartners = null;
+    imStatusTrackedPersonas = Record.many("Persona", {
+        compute() {
+            return Object.values(this.Persona?.records ?? []).filter(
+                (persona) =>
+                    persona.type === "partner" &&
```

Comment:
Aren't we supposed to have im status for guest too now?

Reply:
Apparently it's not supported currently